examples/encryption-example/backup.sh

```
@@ -0,0 +1,438 @@
+}
+
+# Email notification function with enhanced formatting
+send_notification() {
+    if [ -n "$NOTIFY_EMAIL" ]; then
+        local subject="Encrypted MariaDB Backup Failed - $(hostname)"
+        local error_details="$1"
+        local backup_duration="${BACKUP_DURATION:-unknown}"
+        local timestamp
+        timestamp="$(date '+%Y-%m-%d %H:%M:%S')"
+        
+        local message="Encrypted MariaDB backup failed on $(hostname) at $(timestamp).
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)